_data/glossary/certificate.yaml

```diff
@@ -0,0 +1,11 @@
+id: certificate
+name: Certificate
+full-link: https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/
```

Comment:
Should we remove the "https://kubernetes.io" part?

Reply:
Good point, fix in - https://github.com/kubernetes/website/pull/7476